tests/main/snap-confine-from-core/task.yaml

```diff
@@ -34,5 +35,45 @@ execute: |
     get_journalctl_log | MATCH "DEBUG: restarting into"
 
     echo "Ensure snap-confine from the core snap is run"
-    # do not use "strace -f" for unknown reasons that hangs
     test-snapd-tools.echo hello
+
+
+    echo "Check if snap-confine profile generation test is applicable"
+    if [ $(aa-enabled) != "Yes" ]; then
+        if [ "$SPREAD_SYSTEM" = "ubuntu-16.04-64" ]; then
+            echo "No apparmor on ubuntu-16.04-64 is impossible"
+            echo "Test broken"
+            exit 1
+        fi
+        echo "SKIP: no strict confinement"
```

Comment:
The comment is perhaps misleading, I'd suggest `echo "SKIP: apparmor not enabled"`